### PR TITLE
Fetch ADB logs using UDIDs #24

### PR DESCRIPTION
**Pull request by shridharkalagi**

### CODE REVIEW

File: src/main/java/com/thoughtworks/android/AndroidManager.java [hunk 1]
```diff
@@ -106,5 +107,28 @@ public Device getDevice(String udid) throws Exception {
         );
     }
 
+    public void startADBLog(String udid, String filePath) throws Exception {
+        cmd.execForProcessToExecute("adb -s " + udid + " logcat -b all -c");
+        process = cmd.execForProcessToExecute("adb -s " + udid + " logcat > " + filePath);
```
Comment (SrinivasanTarget) [inline src/main/java/com/thoughtworks/android/AndroidManager.java]: @shridharkalagi Have you tried this? Isn't it just for clearing all the buffers? Does it retrieve too?

Comment (saikrishna321) [reply src/main/java/com/thoughtworks/android/AndroidManager.java]: @SrinivasanTarget he is clearing the buffer before creating a new one .

Comment (SrinivasanTarget) [reply src/main/java/com/thoughtworks/android/AndroidManager.java]: command to retrieve?

Comment (saikrishna321) [reply src/main/java/com/thoughtworks/android/AndroidManager.java]: adb -s " + udid + " logcat > " + filePath

Comment (shridharkalagi) [reply src/main/java/com/thoughtworks/android/AndroidManager.java]: "adb -s udid logcat -b all -c"  will clear all the buffers

Comment (shridharkalagi) [reply src/main/java/com/thoughtworks/android/AndroidManager.java]: adb shell logcat -b all -g --to print all the buffers with current buffer size

**Pull request merged**